instance/tests/integration/integration_instance.py

```diff
@@ -379,13 +380,7 @@ def test_spawn_appserver(self):
 
         if settings.DEFAULT_INSTANCE_MYSQL_URL and settings.DEFAULT_INSTANCE_MONGO_URL:
             self.assert_swift_container_provisioned(instance)
-            self.assert_server_ready(instance)
-            self.assert_instance_up(instance)
-            self.assert_appserver_firewalled(instance)
-            self.assertTrue(instance.successfully_provisioned)
```

Comment:
Do we no longer need to check that all of these steps succeeded?

Reply:
These assertions were duplicated when merging integration tests. Those are running 10 lines above.